examples/BasicReading/BasicReading.ino

```diff
@@ -28,11 +28,11 @@
 #define RD_PIN 3
 #define CS_PIN 4
 
-// Data pins D0-D15 (modify these to match your wiring)
-uint8_t dataPins[16] = {40, 41, 37, 36, 35, 34, 39, 38, 24, 25, 14, 15, 23, 20, 21, 16};
+// Data pins D0-D15 are now hardcoded in the library for optimal performance
```

Comment:
Can we just remove these lines instead of commenting them out? And add an explanation to the comment at the top of the file about what pins need to be connected for this library to work?